app/services/book_to_nomis_config.rb

```diff
@@ -1,9 +1,13 @@
 class BookToNomisConfig
-  def initialize(staff_nomis_checker, prison_name, opted_in, already_booked_in_nomis)
-    self.staff_nomis_checker = staff_nomis_checker
-    self.prison_name = prison_name
-    self.opted_in = opted_in
-    self.already_booked_in_nomis = already_booked_in_nomis
+  def initialize(staff_nomis_checker,
+    prison_name, opted_in,
+    already_booked_in_nomis,
+    prisoner_details_presenter)
+    self.staff_nomis_checker        = staff_nomis_checker
```

Comment:
What do you think of using `RequestStore` for these shared objects? It looks like nomis checker, presenter, etc are needed across the request cycle. They are kind of globals already in the view.

An advantage would be that this class could have it's number of constructer arguments trimmed.

This is an idea I've had just now, not really sure if it would actually be a good idea...